styleguide/source/assets/scss/02-molecules/_subscribe-promo.scss

```diff
@@ -0,0 +1,32 @@
+.ama__subscribe-promo {
+  display: inline-block;
+  padding: $gutter / 2;
+  color: $white;
+  background: $purple;
+
+  &__subject-title {
+    font-size: 16px;
+    text-transform: uppercase;
+    padding-bottom: $gutter / 2;
+  }
+
+  .ama__form-group {
+    input {
+      @include type($myriad-pro, $font-weight-regular);
+      font-style: italic;
+    }
+
+    label {
+      visibility: hidden;
+    }
+  }
+
+  button {
```

Comment:
Shouldn't this be using the button.scss styling?

Reply:
If it's a custom button should we add to the button mixins so we can reuse it?

Reply:
This styling does already use the button.scss styling but is also different and so needs to override several of the rules there. I don't know if we'll need to reuse it so I'd rather not change this for now. @labbydev @JaeBea do you have any thoughts about this?

Reply:
This seems like a rather niche case, so I'm fine with not adding a button variant for it.